src/simulations/local_elec.py

```diff
@@ -348,6 +348,10 @@ def step(
             if record:
                 loads_flex = np.zeros(self.n_homes) if self.date == self.date_end \
                     else [sum(batch_flex[home][h][1:]) for home in homes]
+                loaded_buses = self.network.loaded_buses if self.prm['grd']['manage_voltage'] \
+                    else None
+                sgen_buses = self.network.sgen_buses if self.prm['grd']['manage_voltage'] \
+                    else None
```

Comment:
yes! I noticed this problem also. My bad I caused this problem when I debugged the comparison between optimisation and pandapower. I had fixed this in the main branch in parallel.